src/AcceptanceTests/Infrastructure/DefaultServer.cs

```diff
@@ -38,6 +38,7 @@ public Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor,
             var recoverability = configuration.Recoverability();
             recoverability.Delayed(delayed => delayed.NumberOfRetries(0));
             recoverability.Immediate(immediate => immediate.NumberOfRetries(0));
+            configuration.SendFailedMessagesTo("error");
 
             var transportConfig = configuration.UseTransport<MsmqTransport>();
```

Comment:
This brings up a good point:

Are there other downstreams that are using MSMQ for test unnecessary? (yes there are IMO)

Should we convert them as a separate issue or just make that one step when they first adopt the v7 test suites? (since MSMQ is no longer there)

Same goes for InMemory persistence since that is likely to be pulled out of the core as well.

@Particular/nservicebus-maintainers thoughts?